api/samples/opcode_count.cpp

```diff
@@ -67,21 +67,33 @@ static droption_t<int>
            "The opcode to consider when counting the number of times "
            "the instruction is executed. Default opcode is set to add.");
 
+#ifdef SHOW_RESULTS
+static droption_t<bool> show_results_(DROPTION_SCOPE_CLIENT, "show_results", true,
```

Comment:
Hmm, I guess this does still satisfy the expectations of SHOW_RESULTS for exported samples vs the internal tests.  Seems better to have the ifdef only around the true/false default value to avoid duplicating the rest.